data/korath outfits.txt

```diff
@@ -91,6 +91,15 @@ outfit "Large Heat Shunt"
 	"cooling" 54
 	description "Without these powerful cooling systems, most Korath warships would overheat within seconds as soon as they began to fire their weapons."
 
+outfit "Fuel Collector"
+	category "Systems"
+	cost "101000"
+	thumbnail "outfit/unknown"
+	"mass" 20
+	"outfit space" -20
+	"fuel capacity" 200
+	"ramscoop" 0.5
+	description "The Korath Fuel Collector is a large system combining a weak ramscoop with enough fuel capacity for one jump drive jump or two hyperdrive jumps."
```

Comment:
Description should not reference the amount of fuel needed for various jump modes - those value can be changed in mods.

I suggest developing the description from Korath lore, referencing their habitual tendency to plunder civilizations both near and far, etc.

Reply:
Is the new description better?